tests/chainerx_tests/unit_tests/routines_tests/test_creation.py

```diff
@@ -383,12 +348,23 @@ def test_ascontiguousarray_from_chainerx_array(device, shape, dtype, padding):
     a = chainerx.ascontiguousarray(obj)
     if not padding and shape != ():  # () will be reshaped to (1,)
         assert a is obj
-    e = chainerx.ascontiguousarray(np_arr)
+    e = chainerx.asarray(np_arr)
```

Comment:
`np_arr` should just work in `chainerx.testing.assert_array_equal_ex`. How about simply removing this line?

Reply:
I like the variable name so removed the function calling instead!